src/app/component/Minimap/MiniMap.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import MiniMap from './MiniMap';
+
+storiesOf("Components | Button", module)
```

Comment:
We are now trying to write stories the same as Input Component, with Knobs decorator

Reply:
I added knobs decorator, however I noticed when changing inside of storybook, the props do not resize right away. It only resize if you write in stories.js code